packages/api/src/routers/auth/mobile/mobile_auth_router.ts

```diff
@@ -60,5 +61,12 @@ export function mobileAuthRouter() {
     res.status(payload.statusCode).json(payload.json)
   })
 
+  router.post('/android-apple-redirect', (req, res) => {
+    const { id_token } = req.body
+    return res.redirect(
+      `${env.client.url}/android-apple-token?token=${id_token as string}`
```

Comment:
This url will actually never get called. Android WebView has a hook called `shouldOverrideUrlLoading` that we'll use to match this url. Then we parse out the token, dismiss then WebView, then send the apple token to the viewModel.